.github/workflows/test.yml

```diff
@@ -59,3 +66,12 @@ jobs:
       - run: mix format --check-formatted
       - run: mix compile --warnings-as-errors
       - run: mix test --include slow
+
+  format:
+    runs-on: ubuntu-latest 
+    steps:
+      - uses: actions/checkout@v4 
+      - uses: erlef/setup-beam@v1
+        with:
+          elixir-version: "latest"
+      - run: mix format --check-formatted
```

Comment:
Since formatting rules between Elixir versions are incompatible, I'm only using the latest one for the test.